Microsoft.Azure.CosmosRepository/src/Builders/IItemContainerBuilder.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using Microsoft.Azure.CosmosRepository.Options;
+
+namespace Microsoft.Azure.CosmosRepository.Builders
+{
+    /// <summary>
+    /// A builder to configure a container for an item
+    /// </summary>
+    public interface IItemContainerBuilder
+    {
+        /// <summary>
+        ///
```

Comment:
We should probably have something here, especially since it is `public`.

Reply:
Will amend this must have missed it off.